ocean_lib/web3_internal/test/test_utils.py

```diff
@@ -9,12 +9,19 @@
 )
 
 
-def test_generate_multi_value_hash(alice_address):
+def test_generate_multi_value_hash(alice_address, alice_private_key):
```

Comment:
I just tested this, in this branch, and it works now. +1

Reply:
Agreed, and that's what I suggested as well.

Reply:
And @mariacarmina to make the test pass in all instances, just get the value from env instead of hardcoding it. E.g. expected = os.getenv('TEST_PRIVATE_KEY1') instead of "0x6..."

Reply:
Agreed, thank you @DMats and @calina-c.

Reply:
Modified the test keys in pytest.ini to match tox.ini.

Reply:
Also, @DMats in your code snippet, the assert related to `alice_private_key` will fail, because now it's another test key for that (the one from the comment).

Reply:
@mariacarmina can you confirm you ran the quickstart and everything is ok?

Reply:
Done.